scripts/transfer-doc-files.js

```diff
@@ -101,12 +112,15 @@ function copyDocumentationFiles(component) {
  * @param {string} component - the name of the current component.
  * @param {string} readmeFilePath - the absolute path to the readme file
 */
-function saveMarkdownWithFrontMatter(component, readmeFilePath) {
+function saveMarkdownWithFrontMatter(component, readmeFilePath, targetDir, extension = 'md') {
     const frontMatter = frontMatterTemplate(component);
-    const file = fs.readFileSync(readmeFilePath, {encoding: 'utf8'});
+    let file = fs.readFileSync(readmeFilePath, {encoding: 'utf8'});
+    const currentFrontMatter = file.match(/(---\n)(.*)(\n---)/s);

```

Comment:
Could have "cached" the regex and reused the variable.

Reply:
Yes. Changed it.